packages/wallets/package.json

```diff
@@ -18,6 +18,7 @@
         "classnames": "^2.2.6",
         "embla-carousel-react": "^8.0.0-rc12",
         "formik": "^2.1.4",
+        "framer-motion": "^6.5.1",
```

Comment:
We should avoid using framer-motion for such simple animation. Best to use vanilla css. The library is huge (128kb) and we are not using a lot of the features.

```
.animated-list-item {
  height: 0;
  opacity: 0;
  overflow: hidden;
  transition: height 0.2s, opacity 0.15s 0.05s;
}

.animated-list-item.open {
  height: auto;
  opacity: 1;
}
```

Reply:
sure @yashim-deriv will refactor

Reply:
hey @yashim-deriv 
there are my new changes:
1. Instead of `framer-motion` library, i used `react-trasition-group` library which [size](https://bundlephobia.com/package/react-transition-group@4.4.5) is in 10 times less 
2. There are also where challenges with changing the height from 0 to 'auto'. Actually we can't do transition from 0 height to 'auto'. There are some workarounds, but almost all of them have strong disadvantages.
I have applied workaround using display grid property ([article](https://www.stefanjudis.com/snippets/how-to-animate-height-with-css-grid/)). With current implementation animation looks nice

Reply:
Also I would like to know. Does it make it sense to create generic `AnimatedList` component and build `FadedAnimatedList` on top of `AnimatedList`?

**AnimatedList:**

```
const AnimatedList: React.FC<TProps> = ({ animateOnMount = false, animationTimeout = 250, children, className }) => {
    return (
        <TransitionGroup className={className}>
            {children.map(child => {
                return (
                    <CSSTransition
                        appear={animateOnMount}
                        classNames={`${className}__list-item`}
                        key={child.key}
                        timeout={animationTimeout}
                    >
                        {child}
                    </CSSTransition>
                );
            })}
        </TransitionGroup>
    );
};
```

**FadedAnimatedList:**

```
const FadedAnimatedList: React.FC<TProps> = ({ animateOnMount = false, children }) => {
    return (
        <AnimatedList animateOnMount={animateOnMount} animationTimeout={250} className='wallets-faded-animated-list'>
            {children.map(child => {
                return (
                    <div className='wallets-faded-animated-list__list-item--display-grid' key={child.key}>
                        {child}
                    </div>
                );
            })}
        </AnimatedList>
    );
};
```